Sources/Rendering/OpenGL/PolyDataMapper2D/index.js

```diff
@@ -0,0 +1,886 @@
+// import { mat3, mat4, vec3 } from 'gl-matrix';
+import { mat4 } from 'gl-matrix';
+
+import * as macro from 'vtk.js/Sources/macros';
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
```

Comment:
Would it be better to simply `import { round } from vtk.js/Sources/Common/Core/Math` since only round() function is being used.